great_expectations/cli/datasource.py

```diff
@@ -149,7 +211,7 @@ def _add_sqlalchemy_datasource(context):
             context.add_datasource(name=data_source_name, class_name='SqlAlchemyDatasource', **configuration)
             break
         except ModuleNotFoundError as de:
-            message = message + "\n  - Please `pip install psycopg2` and try again"
+            message = message + "\n  - Please `pip install psycopg2` and try again" #TODO: Taylor, this looks wrong (?)
```

Comment:
Agreed. Not sure about the below, but it should be something other than guessing psycopg2

```suggestion
            message = message + "\n  - Unable to load a required module: " + str(de)
```

Reply:
Implemented a temp fix and created a task to improv the error message with the specific module name for 0.8.5